src/main.rs

```diff
@@ -231,6 +231,14 @@ fn create_app<'a, 'b>(
                 .default_value(&api_server_path)
                 .group("vmm-config"),
         )
+        .arg(
+            Arg::with_name("restore-from")
+                .long("restore-from")
+                .help("Restore from a VM snapshot.")
```

Comment:
I think here we should document that the argument for this should be a valid URL, e.g. `file:///foo/bar/` or tcp://192.168.1.10/`